### ISSUE TITLE
Better name

### ISSUE THREAD

**Issue by palash25** [p-society/slackcontestwatcherbot#6]
The current name is quite long.

Lets keep this issue open for suggestions for better names if that's okay with you @bkhanale 

I was thinking something like CP-Sentry ?

**Comment by bkhanale** [comment]
Yeah no problem. Any name will work just fine.

**Comment by piyushmishra12** [comment]
How about 'Confest'?

**Comment by palash25** [comment]
Explain the name please?

**Comment by piyushmishra12** [comment]
Just a portmanteau of contest and fest lol 😝

**Comment by palash25** [comment]
> Just a portmanteau of contest and fest lol stuck_out_tongue_closed_eyes

I got that but why fest :thinking: ? Do you mean` competitive contest fest `something like that?

**Comment by piyushmishra12** [comment]
Yes. That's exactly what I meant. Since it'll give the details of all the upcoming contests. Seems far fetched?

**Comment by palash25** [comment]
Mine was CP-Sentry because CP - stands for competitive programming and Sentry is someone who keeps watch for something specific I think :confused: 

Well we can have some other people comment here and then have a poll on slack and let the best name win

**Comment by palash25** [comment]
Can you think of a few more portmanteau involving programming/CP and bots ?

**Comment by piyushmishra12** [comment]
Can't think of any at the moment. I'll comment once something strikes me.
